packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -65,6 +66,43 @@ def vip_app(container: marathon.Container, network: marathon.Network, host: str,
         raise AssertionError('Unexpected network: {}'.format(network.value))
 
 
+def pod_vip_app(network: marathon.Network, host: str, vip: str):
```

Comment:
Also, can we add port-mapping as a parameter? Would be useful for bridge-mode networking.

Reply:
Added